openmetadata-service/src/main/java/org/openmetadata/service/migration/api/MigrationWorkflow.java

```diff
@@ -12,24 +18,34 @@
 
 @Slf4j
 public class MigrationWorkflow {
-  private final List<MigrationProcess> migrations;
+  private List<MigrationProcess> migrations;
 
+  private final String nativeSQLScriptRootPath;
+  private final ConnectionType connectionType;
+  private final String extensionSQLScriptRootPath;
   private final MigrationDAO migrationDAO;
   private final Jdbi jdbi;
 
   private final boolean forceMigrations;
 
   public MigrationWorkflow(
-      Jdbi jdbi, String nativeSQLScriptRootPath, ConnectionType connectionType, boolean forceMigrations) {
+      Jdbi jdbi,
+      String nativeSQLScriptRootPath,
+      ConnectionType connectionType,
+      String extensionSQLScriptRootPath,
+      boolean forceMigrations) {
     this.jdbi = jdbi;
     this.migrationDAO = jdbi.onDemand(MigrationDAO.class);
     this.forceMigrations = forceMigrations;
+    this.nativeSQLScriptRootPath = nativeSQLScriptRootPath;
+    this.connectionType = connectionType;
+    this.extensionSQLScriptRootPath = extensionSQLScriptRootPath;
+  }
+
+  public void loadMigrations() {
```

Comment:
added a `loadMigrations` to be run separately instead of inside the constructor to ease the testing